benchmarks/jdk-concurrent/src/main/scala/org/renaissance/jdk/concurrent/FjKmeans.scala

```diff
@@ -4,9 +4,12 @@ import org.renaissance.Benchmark
 import org.renaissance.Benchmark._
 import org.renaissance.BenchmarkContext
 import org.renaissance.BenchmarkResult
-import org.renaissance.BenchmarkResult.Validators
+import org.renaissance.BenchmarkResult.Assert
+import org.renaissance.BenchmarkResult.ValidationException
 import org.renaissance.License
 
+import java.{lang => jl, util => ju}
```

Comment:
I don't think this shortening style is used anywhere else in the project.
For code readability, I would have a preference to keep explicit `java.util` and `java.lang` in code. Especially since we're not saving that many characters here.